swagger.yaml

```diff
@@ -0,0 +1,137 @@
+---
+swagger: "2.0"
+info:
+  description: "Student Backend Portal"
+  version: "0.0.1"
+  title: "Portal System"
+host: "localhost:8080"
+basePath: "/"
+schemes:
+  - "http"
+consumes:
+  - "application/json"
+produces:
+  - "application/json"
+paths:
+  /student:
+    post:
+      operationId: "addStudent"
+      parameters:
+        - in: "body"
+          name: "student"
+          description: "student's details"
+          required: false
+          schema:
+            $ref: "#/definitions/student"
+      responses:
+        201:
+          description: "student added"
+          schema:
+            $ref: "#/definitions/student"
+        409:
+          description: "student already exists"
+        500:
+          description: "internal server error"
+
+  /listStudent:
+    get:
+      operationId: "ListStudents"
+      parameters:
+        - in: "body"
```

Comment:
We cannot have body in get calls. You can use schema for "in query". You will need to have in query params for each key like name age etc.